### PR TITLE
refactor: tanya`s points

### PR DESCRIPTION
**Pull request by tanyaivattt**

### CODE REVIEW

File: src/components/cardOfCourse/_tests_/cardOfCourse.spec.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { render, fireEvent, screen } from '@testing-library/react';
 import { MemoryRouter } from 'react-router-dom';
 import '@testing-library/jest-dom';
-import CardOfCourse from '../CardOfCourse';
+import {CardOfCourse} from '../cardOfCourse';
```
Comment (Nazaziik) [inline src/components/cardOfCourse/_tests_/cardOfCourse.spec.tsx:4]: bracket spaces: "{ CardOfCourse }" ?

File: src/components/instructorCard/_instructorCard.module.scss [hunk 2]
```diff
@@ -34,18 +18,21 @@ $font-nunito: 'Nunito', sans-serif;
...
   }
+
   & .info-container {
     display: flex;
     flex-direction: column;
     align-items: center;
     gap: 8px;
+
     & .img-container {
       & img {
```
Comment (Nazaziik) [inline src/components/instructorCard/_instructorCard.module.scss:37]: maybe add 1 more empty line ? (suggestion)

File: src/pages/instructorsPage/instructorsPage.tsx [hunk 3]
```diff
@@ -4,7 +4,7 @@ import { useAppDispatch, useAppSelector } from '../../redux/store/hooks';
 import HeaderInstructors from '../../components/headerInstructors/headerInstructors';
 import { getInstructors } from '../../redux/instructors/instructorsAction';
 import {filterByCategory, searchInstructors} from '../../redux/instructors/instructorsSlice';
-import InstructorCard from '../../components/instructorCard/instructorCard';
+import {InstructorCard} from '../../components/instructorCard/instructorCard';
```
Comment (Nazaziik) [inline src/pages/instructorsPage/instructorsPage.tsx:7]: again: "{ InstructorCard }"

File: src/pages/instructorsPage/instructorsPage.tsx [hunk 4]
```diff
@@ -4,7 +4,7 @@ import { useAppDispatch, useAppSelector } from '../../redux/store/hooks';
 import HeaderInstructors from '../../components/headerInstructors/headerInstructors';
 import { getInstructors } from '../../redux/instructors/instructorsAction';
 import {filterByCategory, searchInstructors} from '../../redux/instructors/instructorsSlice';
```
Comment (Nazaziik) [inline src/pages/instructorsPage/instructorsPage.tsx:6]: and here: "{ filterByCategory, searchInstructors }"

File: src/pages/instructorsPage/instructorsPage.tsx
Comment (Nazaziik) [inline src/pages/instructorsPage/instructorsPage.tsx:1]: If You are doing that change to fix error, maybe It is good idea to refactor whole file and inform responsible person ?
(just asking)

File: src/redux/instructors/instructorsSlice.ts
Comment (Nazaziik) [inline src/redux/instructors/instructorsSlice.ts:1]: 1 extra empty line (line 4 or 5)

Comment (Nazaziik) [reply src/redux/instructors/instructorsSlice.ts:1]: Why do we have 2 exports in 1 file ?
(I want to be sure that SRP is not broke and that is the best solution)

File: src/components/instructorCard/_instructorCard.module.scss (hunk 2 above)
Comment (tanyaivattt) [reply src/components/instructorCard/_instructorCard.module.scss:37]: ok, i added

File: src/pages/instructorsPage/instructorsPage.tsx
Comment (tanyaivattt) [reply src/pages/instructorsPage/instructorsPage.tsx:1]: ok, it will be right

**Review by Nazaziik — DISMISSED** [review]

**Review by VitalikMatvieiev — DISMISSED** [review]

**Review by Virsta — APPROVED** [review]

**Review by Nazaziik — APPROVED** [review]

**Pull request merged**